src/components/pages/UserManagement/UserManagement.js

```diff
@@ -110,11 +91,21 @@ const UserManagement = ({ allMentors, allMentees }) => {
   return (
     <>
       <h2>Manage Users</h2>
-      <Switch
-        checkedChildren={`${displayRole}`}
-        unCheckedChildren={`${displayRole}`}
+      <Tabs
```

Comment:
transitions are smooth and code is dry. simple and no fluff. thanks for taking the time to style it